### PR TITLE
Amend rhobs SLO latencies Dashboards with renegotiated latency SLOs

### PR DESCRIPTION
**Pull request by moadz**
* 3s => 5s for 1m sample SLO
* 10s => 15s for 10m sample SLO
* 20s => 100s for 100m sample SLO

Signed-off-by: mzardab <mzardab@redhat.com>

### CODE REVIEW

File: observability/dashboards/slo.libsonnet [hunk 1]
```diff
@@ -441,28 +441,28 @@ function(instanceName, environment, dashboardName) {
...
       'sum(rate(up_custom_query_duration_seconds_count{namespace="%s",query="query-path-sli-1M-samples"}[28d]))' % instance.upNamespace,
       6
     ) +
     latencyRow(
-      '90% of valid requests that process 10M samples return < 10s',
+      '90% of valid requests that process 10M samples return < 20s',
       0.9,
       10,
-      'sum(rate(up_custom_query_duration_seconds_bucket{namespace="%s",query="query-path-sli-10M-samples",le="10.761264004567169"}[28d]))' % instance.upNamespace,
+      'sum(rate(up_custom_query_duration_seconds_bucket{namespace="%s",query="query-path-sli-10M-samples",le="15"}[28d]))' % instance.upNamespace,
```
Comment (jessicalins) [inline observability/dashboards/slo.libsonnet:456]: shouldn't here be `le=20` ?

File: resources/observability/grafana/observatorium/grafana-dashboard-slo-mst-production.configmap.yaml [hunk 2]
```diff
@@ -993,7 +993,7 @@ data:
                 "targets": [
                     {
                         "exemplar": true,
-                        "expr": "clamp_min(\n    (\n        (\n            sum(rate(up_custom_query_duration_seconds_bucket{namespace=\"observatorium-mst-production\",query=\"query-path-sli-10M-samples\",le=\"10.761264004567169\"}[28d]))\n            /\n            sum(rate(up_custom_query_duration_seconds_count{namespace=\"observatorium-mst-production\",query=\"query-path-sli-10M-samples\"}[28d]))\n        ) - 0.90000000000000002\n    )\n    /\n    (1 - 0.90000000000000002), 0)\n",
+                        "expr": "clamp_min(\n    (\n        (\n            sum(rate(up_custom_query_duration_seconds_bucket{namespace=\"observatorium-mst-production\",query=\"query-path-sli-10M-samples\",le=\"15\"}[28d]))\n            /\n            sum(rate(up_custom_query_duration_seconds_count{namespace=\"observatorium-mst-production\",query=\"query-path-sli-10M-samples\"}[28d]))\n        ) - 0.90000000000000002\n    )\n    /\n    (1 - 0.90000000000000002), 0)\n",
```
Comment (jessicalins) [inline resources/observability/grafana/observatorium/grafana-dashboard-slo-mst-production.configmap.yaml:996]: also here

File: observability/dashboards/slo.libsonnet (hunk 1 above)
Comment (jessicalins) [reply observability/dashboards/slo.libsonnet:456]: or in line 435 `...samples return < 15s`

File: resources/observability/grafana/observatorium/grafana-dashboard-slo-mst-stage.configmap.yaml [hunk 3]
```diff
@@ -993,7 +993,7 @@ data:
                 "targets": [
                     {
                         "exemplar": true,
-                        "expr": "clamp_min(\n    (\n        (\n            sum(rate(up_custom_query_duration_seconds_bucket{namespace=\"observatorium-mst-stage\",query=\"query-path-sli-10M-samples\",le=\"10.761264004567169\"}[28d]))\n            /\n            sum(rate(up_custom_query_duration_seconds_count{namespace=\"observatorium-mst-stage\",query=\"query-path-sli-10M-samples\"}[28d]))\n        ) - 0.90000000000000002\n    )\n    /\n    (1 - 0.90000000000000002), 0)\n",
+                        "expr": "clamp_min(\n    (\n        (\n            sum(rate(up_custom_query_duration_seconds_bucket{namespace=\"observatorium-mst-stage\",query=\"query-path-sli-10M-samples\",le=\"15\"}[28d]))\n            /\n            sum(rate(up_custom_query_duration_seconds_count{namespace=\"observatorium-mst-stage\",query=\"query-path-sli-10M-samples\"}[28d]))\n        ) - 0.90000000000000002\n    )\n    /\n    (1 - 0.90000000000000002), 0)\n",
```
Comment (jessicalins) [inline resources/observability/grafana/observatorium/grafana-dashboard-slo-mst-stage.configmap.yaml:996]: and here

File: resources/observability/grafana/observatorium/grafana-dashboard-slo-telemeter-stage.configmap.yaml [hunk 4]
```diff
@@ -1327,7 +1327,7 @@ data:
                 "targets": [
                     {
                         "exemplar": true,
-                        "expr": "clamp_min(\n    (\n        (\n            sum(rate(up_custom_query_duration_seconds_bucket{namespace=\"observatorium-stage\",query=\"query-path-sli-10M-samples\",le=\"10.761264004567169\"}[28d]))\n            /\n            sum(rate(up_custom_query_duration_seconds_count{namespace=\"observatorium-stage\",query=\"query-path-sli-10M-samples\"}[28d]))\n        ) - 0.90000000000000002\n    )\n    /\n    (1 - 0.90000000000000002), 0)\n",
+                        "expr": "clamp_min(\n    (\n        (\n            sum(rate(up_custom_query_duration_seconds_bucket{namespace=\"observatorium-stage\",query=\"query-path-sli-10M-samples\",le=\"15\"}[28d]))\n            /\n            sum(rate(up_custom_query_duration_seconds_count{namespace=\"observatorium-stage\",query=\"query-path-sli-10M-samples\"}[28d]))\n        ) - 0.90000000000000002\n    )\n    /\n    (1 - 0.90000000000000002), 0)\n",
```
Comment (jessicalins) [inline resources/observability/grafana/observatorium/grafana-dashboard-slo-telemeter-stage.configmap.yaml:1330]: and here

File: observability/dashboards/slo.libsonnet (hunk 1 above)
Comment (moadz) [reply observability/dashboards/slo.libsonnet:456]: Good catch!

**Review by jessicalins — APPROVED** [review]
LGTM :star2:

**Pull request merged**